config/extinctionCoeffs.py

```diff
@@ -2,13 +2,17 @@
 # Numbers provided by Masayuki Tanaka (NAOJ).
 #
 # Band, A_filter/E(B-V)
+
+# These default extinction coefficients are WRONG, but better than nothing
```

Comment:
I think that all-caps WRONG is a bit strong here.  "These default extinction coefficients are not optimal for the FGCM standard bandpasses.  TO DO..."